app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+
+  belongs_to :author, class_name: 'User'
+  after_save :update_posts_counter
+
+  def return_recent_comments
+    Comment.order(created_at: :desc).limit(5)
```

Comment:
- [ ] this should be `comments.order(create_at: :desc).limit(5)`, kindly fix this